README.md

```diff
@@ -1,7 +1,25 @@
-<h2>This is a fully functional ToDo List app made using NodeJs , Express & MongoDB Atlas</h2>
+* This is a fully functional ToDo List app made using 
+    * NodeJs 
+    * Express 
+    * MongoDB
+    * Atlas
```

Comment:
Its a single point ... MongoDB Atlas

Reply:
oh I mistook that and😅 will change that now